TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/AbstractNodeData.php

```diff
@@ -177,7 +178,7 @@ public function setProperty($propertyName, $value)
                 return;
             }
 
-            $this->properties[$propertyName] = $value;
+            $this->properties[$propertyName] = is_string($value) ? Utility::removeControlCharactersFrom($value) : $value;
```

Comment:
![bildschirmfoto 2017-05-19 um 09 48 25](https://cloud.githubusercontent.com/assets/33351/26238226/77286c9c-3c78-11e7-9f7f-d7f13eeef736.png)
Mmmh, but why doesn't it escape these chars? In a simple test `json_encode` handles all characters pretty well.